e2e/commands/start/start_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package start_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ZupIT/horusec/internal/utils/testutil"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("running binary Horusec with start parameter", func() {
+	var (
+		session           *gexec.Session
+		err               error
+		flags             map[string]string
+		repoAuthorization string
+	)
+
+	JustBeforeEach(func() {
+		cmd := testutil.GinkgoGetHorusecCmdWithFlags(testutil.CmdStart, flags)
+		session, err = gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
+
+	})
+
+	When("--project-path is passed", func() {
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath: testutil.GoExample1,
+			}
+		})
+
+		It("Then the following validations are performed", func() {
```

Comment:
"Then the following validations are performed" is a good step name? I think that this name is not a bit declarative

Reply:
Adjusted!